R/pr.R

```diff
@@ -80,26 +80,23 @@
 #'
 #' @section Overview of all the functions:
 
-#' * `pr_init()`: Does a preparatory pull of the default branch from the source
-#' repo, to get a good start point. Creates and checks out a new branch. Nothing
-#' is pushed to or created on GitHub (that does not happen until the first time
-#' you call `pr_push()`).
+#' * `pr_init()`: As a contributor, start work on a new PR by ensuring that
+#' your local repo is up-to-date, then creating and checking out a new branch.
+#' Nothing is pushed to or created on GitHub until you call `pr_push()`.
+
+#' * `pr_fetch()`: As a maintainer, review or contribute changes to an existing
+#' PR by creating a local branch that tracks the remote PR. `pr_fetch()` does as
+#' little work as possible, so you can also use it to resume work on an PR that
+#' already has a local branch (where it will also ensure your local branch is
+#' up-to-date). If called with no arguments, up to 9 open PRs are offered for
+#' interactive selection.
 
 #' * `pr_resume()`: Resume work on a PR by switching to an existing local branch
 #' and pulling any changes from its upstream tracking branch, if it has one. If
 #' called with no arguments, up to 9 local branches are offered for interactive
 #' selection, with a preference for branches connected to PRs and for branches
 #' with recent activity.
```

Comment:
Should we mention that `pr_resume()` can be useful if you've used `pr_pause()` but have not yet called `pr_push()`? But otherwise we generally use `pr_fetch()` because you can reference by PR number and that is typically a little easier?

Reply:
I added some text to `pr_pause()` to hopefully help here.